dev/validate_bom__test.py

```diff
@@ -861,9 +946,35 @@ def make_test_command_or_none(self, test_name, spec, metric_labels):
         'python', test_path,
         '--log_dir', citest_log_dir,
         '--log_filebase', test_name,
-        '--native_host', 'localhost',
-        '--native_port', str(self.__forwarded_ports[microservice_api].port)
     ]
+    if microservice_api in self.__outbound_service_configs:
+      service_config = self.__outbound_service_configs[microservice_api]
+      host = service_config['host']
+      command.extend([
+          '--native_host', host
+      ])
+      if host.startswith('http://'):
```

Comment:
This is odd.
Why not either have a service_config['host'], service_config['port'], service_config['protocol'] or a service_config['base_url']? If the service_config['port'] is not defined then you coudl default it based on the protocol.